public/app/core/constants.ts

```diff
@@ -12,3 +14,18 @@ export const LS_PANEL_COPY_KEY = 'panel-copy';
 export const PANEL_BORDER = 2;
 
 export const EDIT_PANEL_ID = 23763571993;
+
+export const PAGE_SIZES: Array<SelectableValue<number>> = [
```

Comment:
IMO this const can go to `Pagination.constants.ts`

Reply:
Done